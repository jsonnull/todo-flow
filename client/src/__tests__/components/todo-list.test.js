import React from 'react';
import renderer from 'react-test-renderer';

import TodoList from '../../components/todo-list';

describe('TodoList', () => {
  it('renders without crashing', () => {
    const component = renderer.create(<TodoList />);
    expect(component).toMatchSnapshot();
  });
});
