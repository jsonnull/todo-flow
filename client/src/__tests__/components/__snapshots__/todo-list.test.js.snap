exports[`TodoList renders without crashing 1`] = `<ul />`;
